### ISSUE TITLE
require_full_tree does not work for assets in a child plugin.

### ISSUE THREAD

**Issue by kevana** [bertramdev/asset-pipeline#64]
Files requested individually are loaded but those requested with require_full_tree are not.
@davydotcom mentioned that this wont work its due to a class loader limitation

Parent Project has no assets overridden, child structure:
```
child-project/
    grails-app/
         assets/
             javascripts/
                 application.js
                 core/
                     <multiple nested directories>
```
application.js

    //= require_full_tree core

**Comment by davydotcom** [comment]
Working on a fix for this for next release

**Comment by kevana** [comment]
Thanks!

**Comment by davydotcom** [comment]
Fixed in 2.8.0 release (3.2.0 grails3)
